DatadogSessionReplay/Tests/Processor/ProcessorTests.swift

```diff
@@ -171,6 +188,11 @@ class ProcessorTests: XCTestCase {
             XCTAssertEqual(enrichedRecord.sessionID, expectedRUM.ids.sessionID)
             XCTAssertEqual(enrichedRecord.viewID, expectedRUM.ids.viewID)
         }
+
+        XCTAssertEqual(
+            (core.context.featuresAttributes["session-replay"]?.attributes["records_count"] as? [String: Int64])?.values.map { $0 },
+            [4, 4]
+        )
```

Comment:
suggestion/ This becomes hard to read and repeats few times in this PR. How about creating convenience extension for asserting SR context in tests? With SR context becomes more and more centric to this product, it feels like good investment.

Example:
```swift
XCTAssertEqual(core.context.featureAttributes.recordsCount.values, [4, 4])
```

Reply:
Either that or make a good use of type inferring and dynamic lookup which `FeatureBaggage` support, here it would be:
```swift
let records_count: [String: Int64]? = core.context.featuresAttributes["session-replay"]?.records_count
XCTAssertEqual(records_count?.values.map { $0 }, [4, 4])
```